src/controls/dev/TitleBar/TitleBar_themeresources.xaml

```diff
@@ -224,7 +224,7 @@
                                 </VisualState>
                                 <VisualState x:Name="Disabled">
                                     <Storyboard>
-                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="Content" Storyboard.TargetProperty="Background">
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="RootGrid" Storyboard.TargetProperty="Background">
```

Comment:
Is it a good idea to be setting the template-bound background property here? That feels a little wrong to me. Would it be better to be setting the background on a content control around the animated icon?

Reply:
Ignore that. I see that pattern used elsewhere